pkg/udf/forward/forward.go

```diff
@@ -476,7 +478,7 @@ func (isdf *InterStepDataForward) streamMessage(ctx context.Context, dataMessage
 		return nil, fmt.Errorf("failed to applyUDF, error: %w", err)
 	}
 
-	metrics.UDFProcessingTime.With(metricLabels).Observe(float64(time.Since(start).Microseconds()))
+	metrics.ConcurrentUDFProcessingTime.With(metricLabels).Observe(float64(time.Since(start).Microseconds()))
```

Comment:
Neither of these metrics makes sense to stream mode, since we don't have a way to exclude the buffer writing time.

Reply:
> we don't have a way to exclude the buffer writing time.

Yes, I know that. My idea is: how about redefining ConcurrentUDFProcessingTime as it includes both udf processing time and writing time to buffers. I described that in the metrics document, in the third commit (doc: relationship between...).

Reply:
> In Rust we are only using udf_processing_time.

I never knew that, thanks. I rethink it is better to use `UDFProcessingTime` in Golang. I'll update this PR.